docs-source/content/security/secrets.md

```diff
@@ -2,26 +2,26 @@
 title: "Secrets"
 date: 2019-02-23T17:36:33-05:00
 weight: 6
-description: "Kubernetes secrets for the WebLogic Kubernetes Operator"
```

Comment:
It looks like this section is missing MII specifics (its runtime secret, its model secrets), and OPSS wallet secrets.  I assume this isn't a show-stopper.

Reply:
can you point me at the stuff you think is missing?

Reply:
configuration.model.runtimeEncryptionSecret - required for MII - must stay same for life of a domain runtime - but can change between whole-domain-restarts: 
 /userguide/managing-domains/model-in-image/usage.md#required-runtime-encryption-secret, 

configuration.opss.walletPasswordSecret, configuration.opss.walletFileSecret (also mentioned in usage).